smac/optimizer/multi_objective/aggregation_strategy.py

```diff
@@ -9,22 +11,24 @@
 
 class AggregationStrategy(AbstractMultiObjectiveAlgorithm):
     """
-    An abstract class to aggregate multi-objective losses to a single objective losses, which can then be utilized
-    by the single-objective optimizer.
+    An abstract class to aggregate multi-objective losses to a single objective losses,
```

Comment:
[...] losses to a scalar loss.

If you do not consider the change, then at least change "single objective losses" to "... loss" i.e. singular